### PR TITLE
Rename image raw

### LINKED ISSUE #123: Rename image_raw message
@SammyRamone mentioned, we should rename our `image_raw` topic to something like `image_proc` since it is not raw anymore...
For backwards compatibility (e.g. for rosbags), we could write a little script, which remaps the topic.

### PR DESCRIPTION
**Pull request by jaagut**
## Proposed changes
Changes image topic from `image_raw` to `image_proc`

## Related issues
Closes #123.

## Necessary checks
- [x] Update package version
- [x] Run `catkin build`
- [x] Write documentation
- [x] Create issues for future work
- [x] Test on your machine
- [x] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

**Comment by JanGut** [comment]
@hendrikbrandt Is this still a work in progress or is it ready to review?

File: bitbots_vision/config/visionparams.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ ball_fcnn_candidate_refinement_iteration_count: 1  # Number of iterations of ref
 ball_fcnn_publish_output: false  # Publish the output of the ball fcnn as ImageWithRegionOfInterest
 ball_fcnn_publish_field_boundary_offset: 5  # The offset added to the field_boundary when cropping the fcnn output for publication in pixels
 
-neural_network_type: 'yolo_ncs2'  # The neural network type that should be used (fcnn, yolo_opencv, yolo_darknet, yolo_ncs2 or dummy)
+neural_network_type: 'yolo_opencv'  # The neural network type that should be used (fcnn, yolo_opencv, yolo_darknet, yolo_ncs2 or dummy)
```
Comment (jaagut) [inline bitbots_vision/config/visionparams.yaml:40]:
```suggestion
neural_network_type: 'yolo_ncs2'  # The neural network type that should be used (fcnn, yolo_opencv, yolo_darknet, yolo_ncs2 or dummy)
```

File: bitbots_vision/docs/index.rst [hunk 2]
```diff
@@ -201,6 +201,18 @@ In the bitbots_vision package, special tools for debugging/introspection purpose
    -  ``2`` -> Move current image to **Balls** subdirectory
    -  ``3`` -> Move current image to **Goals** subdirectory
 
+- ``/scripts/rosbag_remapper.py``
+
+   This script remaps the "image_raw" topic in old rosbags to the new "camera/image_proc". Input can either be a single
+   bag or a folder containing bags. The new output bag or folder has an "_updated" appended to its name.
```
Comment (jaagut) [inline bitbots_vision/docs/index.rst:207]:
```suggestion
   This script remaps the ``image_raw`` topic in old rosbags to the new ``camera/image_proc``. Input can either be a single
   bag or a folder containing bags. The new output bag or folder has an ``_updated`` appended to its name.
```

File: bitbots_vision/docs/index.rst [hunk 3]
```diff
@@ -201,6 +201,18 @@ In the bitbots_vision package, special tools for debugging/introspection purpose
...
+
+   Usage:
+
+   ::
+
+    ./rosbag_remapper.py /path/to/rosbag_or_folder
```
Comment (jaagut) [inline bitbots_vision/docs/index.rst:213]:
```suggestion
    rosrun bitbots_vision rosbag_remapper.py /path/to/rosbag_or_folder
```

File: bitbots_vision/scripts/rosbag_remapper.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+A script that remaps the /image_raw topic in rosbags to /camera/image_proc.
+
+Usage:
+python rosbag_remapper.py [input file/directory]
+"""
+
+
+def remap(input_bag_path, output_bag_path):
+    """
+    remaps the topics of a single bag
```
Comment (jaagut) [inline bitbots_vision/scripts/rosbag_remapper.py:18]:
```suggestion
    Remaps the topics of a single bag
```

File: bitbots_vision/config/visionparams.yaml [hunk 5]
```diff
@@ -3,6 +3,7 @@
 vision_parallelize: true  # Run the neural net and the conventional part in parallel
 vision_blind_threshold: 30  # Brightness threshold under which the vision thinks, that someone forgot the camera cap
 
+ROS_audio_msg_topic: 'speak'  # ROS topic of the audio message
```
Comment (Flova) [inline bitbots_vision/config/visionparams.yaml:6]: Why is speak added here?

Comment (hendrikbrandt) [reply bitbots_vision/config/visionparams.yaml:6]: > Why is speak added here?

I don't know but that was merged from the current master

**Review by timonegk — APPROVED** [review]
Looks good to me, I just fixed the package versions (vision was not increased after merging the master, white balancer did not reset patch version to zero after increasing minor version).

**Review by Flova — APPROVED** [review]

**Pull request merged**